src/lib/components/consignments/__stories__/consignments-todo.story.tsx

```diff
@@ -30,38 +39,32 @@ const metadata: ConsignmentMetadata = {
   displayString: "5/5",
 }
 
-const withMetadata: ConsignmentSetup = {
+const withMetadata = {
   ...withPhotos,
   metadata,
 }
 
-const withLocation: ConsignmentSetup = {
+const withLocation = {
   ...withMetadata,
   location: "Huddersfield, UK",
 }
 
-const withProvenance: ConsignmentSetup = {
+const withProvenance = {
   ...withLocation,
   provenance: "This work has seen many hands.",
 }
```

Comment:
I used to keep these in for safety

```diff
- : ConsignmentSetup
```

Now I don't need to, due to the key-value trick up in `interface States`